karma.conf.js

```diff
@@ -38,29 +39,30 @@ module.exports = function (karma) {
       require('karma-super-dots-reporter'),
       require('karma-mocha-reporter'),
       require('karma-jasmine'),
-      require('karma-chrome-launcher')
+      require('karma-chrome-launcher'),
     ],
 
     webpack: {
       devtool: 'eval',
 
       resolve: {
         modules: ['node_modules'],
-        extensions: ['.js', '.ts']
+        extensions: ['.js', '.ts'],
       },
 
       module: {
         rules: [
           {
             test: /\.ts$/,
-            loader: "ts-loader",
+            loader: 'ts-loader',
             options: {
-              configFile: "test/tsconfig.json",
-            }
-          }
-        ]
+              configFile: 'test/tsconfig.json',
+            },
+          },
+        ],
       },
 
+      plugins: [new webpack.ContextReplacementPlugin(/angular(\\|\/)core(\\|\/)fesm5/, __dirname + '/src')],
```

Comment:
What does this do?

Reply:
It's to prevent webpack warnings like this when running the tests:
```
WARNING in ./node_modules/@angular/core/fesm5/testing.js
1543:19-34 Critical dependency: the request of a dependency is an expression

WARNING in ./node_modules/@angular/core/fesm5/core.js
5569:15-36 Critical dependency: the request of a dependency is an expression

WARNING in ./node_modules/@angular/core/fesm5/core.js
5581:15-102 Critical dependency: the request of a dependency is an expression
```